packages/node-cli/src/metadata/SdkCommandsMetadataPatch.json

```diff
@@ -42,5 +35,26 @@
 				"description": "Specify the file cabinet path of the file to upload. For example, \"/SuiteScripts/file.js\"."
 			}
 		}
-	}
+	},
+	"project:deploy": {
+		"options": {
+			"dryrun": {
+				"name": "dryrun",
+				"option": "dryrun",
+				"description": "Run a preview of your deploy process. Your project is not deployed.",
+				"mandatory": false,
+				"type": "FLAG",
+				"usage": "",
+				"defaultOption": false,
+				"disableInIntegrationMode": false
+			}
+		}
+	},
+	"object:import": {
```

Comment:
Not very important comment but just wanted to say.
I know that  SdkCommandsMetadata is not ordered alphabetically because it is autogenerated from the Java code, but I think it's worth it that this file, SdkCommandsMetadataPatch, that we create manually, is filled in alphabetical order.